pango/src/language.rs

```diff
@@ -119,29 +31,30 @@ impl Language {
         }
     }
 
-    #[doc(alias = "get_sample_string")]
-    #[doc(alias = "pango_language_get_sample_string")]
-    pub fn sample_string(&self) -> GString {
-        unsafe { from_glib_none(ffi::pango_language_get_sample_string(self.to_glib_none().0)) }
-    }
-
     #[cfg(any(feature = "v1_48", feature = "dox"))]
     #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_48")))]
     #[doc(alias = "get_preferred")]
     #[doc(alias = "pango_language_get_preferred")]
-    pub fn preferred(&self) -> Vec<Language> {
+    pub fn preferred() -> Vec<Self> {
```

Comment:
This was always wrong, should we keep the old function signature even in this case?

Reply:
It used to take &self when it wasn't needed...

Reply:
Ah... yes. Seems fine, it wouldn't have been usable for its purpose before.